OO-VehicleTests/FISPlaneSpec.m

```diff
@@ -21,52 +21,46 @@
         plane = [[FISPlane alloc] init];
     });
     
-    it(@"is an FISPlane", ^{
-        expect(plane).toNot.beNil();
-        expect(plane).to.beInstanceOf([FISPlane class]);
-    });
-    
     it(@"is a subclass of FISVehicle", ^{
-        expect([plane class]).to.beSubclassOf([FISVehicle class]);
-    });
-    
-    it(@"has a weight of 255000", ^{
-        expect(plane.weight).to.equal(255000.0f);
-    });
-    
-    it(@"has a top speed of 614", ^{
-        expect(plane.topSpeed).to.equal(614.0f);
-    });
-    
-    it(@"has a current speed of 0",^{
-        expect(plane.currentSpeed).to.equal(0.0f);
-    });
-    
-    it(@"has a current direction of 0 (north)",^{
-        expect(plane.currentDirection).to.equal(0.0f);
+        expect(plane).to.beKindOf([FISVehicle class]);
```

Comment:
used it; tests would fail. docs say "passes if the class Foo is a subclass of the class Bar or if it is identical to the class Bar. Use beKindOf() for class clusters."